### ISSUE TITLE
[Broken] sciencechannel.com JSON Error

### ISSUE THREAD

**Issue by TheGrimKeeper** [yt-dlp/yt-dlp#801]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running yt-dlp version **2021.08.10**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
yt-dlp -v --cookies cookies.txt https://www.sciencechannel.com/video/impossible-engineering-science/scotlands-super-bridge
[debug] Command-line config: ['-v', '--cookies', 'cookies.txt', 'https://www.sciencechannel.com/video/impossible-engineering-science/scotlands-super-bridge']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] yt-dlp version 2021.08.10 (zip)
[debug] Python version 3.8.5 (CPython 64bit) - Linux-5.8.0-7630-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4
[debug] Optional libraries: keyring, mutagen, pycryptodome, sqlite
[debug] Proxy map: {}
[debug] [ScienceChannel] Extracting URL: https://www.sciencechannel.com/video/impossible-engineering-science/scotlands-super-bridge
[debug] Using fake IP 6.28.169.193 (US) as X-Forwarded-For
[ScienceChannel] impossible-engineering-science/scotlands-super-bridge: Downloading JSON metadata
ERROR: Unable to download JSON metadata: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 682, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 3190, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.8/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)


```

## Description

Getting a JSON error when trying to download from sciencechannel.com.  This was working with the latest version for the last couple of weeks after the site redesign but it seems to be broken again.

**Comment by pukkandan** [comment]
@Sipherdrakon - I believe you were the one who added science channel

**Comment by Sipherdrakon** [comment]
I just downloaded that episode yesterday and then just tried this same url to post verbose and it ran just fine. I'm thinking cookie issue but I think that would give you a 400 error and not 404. But with a 404 i am pretty sure that means that it didn't generate a json for you to parse. So still thinking it's not a yt-dlp error.

```
ytdl --cookies-from-browser chrome "https://www.sciencechannel.com/video/impossible-engineering-science/scotlands-super-bridge" --simulate --verbose
[debug] Command-line config: ['--cookies-from-browser', 'chrome', 'https://www.sciencechannel.com/video/impossible-engineering-science/scotlands-super-bridge', '--simulate', '--verbose']
[Cookies] Extracting cookies from chrome
[debug] extracting from: "C:\Users\Sipherdrakon\AppData\Local\Google\Chrome\User Data\Default\Cookies"
[Cookies] Extracted 3418 cookies from chrome
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] yt-dlp version 2021.08.10 (exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.18363-SP0
[debug] exe versions: ffmpeg 4.4-full_build-www.gyan.dev, ffprobe 4.4-full_build-www.gyan.dev
[debug] Optional libraries: mutagen, pycryptodome, sqlite, websockets
[debug] Proxy map: {}
[debug] [ScienceChannel] Extracting URL: https://www.sciencechannel.com/video/impossible-engineering-science/scotlands-super-bridge
[debug] Using fake IP 6.19.65.45 (US) as X-Forwarded-For
[ScienceChannel] impossible-engineering-science/scotlands-super-bridge: Downloading JSON metadata
[ScienceChannel] 3144736: Downloading JSON metadata
[ScienceChannel] impossible-engineering-science/scotlands-super-bridge: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 3144736: Downloading 1 format(s): hls-5281
```

**Comment by TheGrimKeeper** [comment]
@Sipherdrakon - Thank you for checking on this.  Not exactly sure what's going on here.  I tried the download again on the same machine (Linux) and still the same error.  Moved over to my Windows machine and still the same error with this episode.  Then I tried a different episode from the same season and it worked.  I moved back to this episode, and it worked.

I remember a similar JSON 404 problem on Youtube earlier this year that was intermittent, and it seemed to be server side.  So I'm guessing it's the same situation here.  Might be worth watching.  Thanks again.
